source/common/runtime/runtime_features.cc

```diff
@@ -95,6 +95,7 @@ RUNTIME_GUARD(envoy_reloadable_features_use_http3_header_normalisation);
 RUNTIME_GUARD(envoy_reloadable_features_validate_connect);
 RUNTIME_GUARD(envoy_reloadable_features_validate_grpc_header_before_log_grpc_status);
 RUNTIME_GUARD(envoy_reloadable_features_validate_upstream_headers);
+RUNTIME_GUARD(envoy_restart_features_quic_send_server_preferred_address_to_all_clients);
```

Comment:
I strongly disagree with making it default to false. This should always be sent if envoy is configured with SPA.

Reply:
I added the runtime guard to changelog and PR description.

Reply:
The Envoy new code could trigger client behavioral change (client to parse preferred_address transport param). If there are issues in the client implementation, it will be really hard for Envoy users to associate the increase of a client issue with a Envoy rollout, and if it causes client app error, it will be even worse.

Reply:
But by that argument we should runtime guard EVERY change to what we send to clients, including every QUICHE version upgrade that in any way changes what we write on the wire. 

This is an rfc-specified transport parameter; if the envoy operator doesn't want to send it to clients, then they should just not configure SPA in their envoy config. And this is not configured by default, so it is only operators who specifically asked for SPA that send it to clients.

Reply:
Could you also add a similar comment `Ignore the automated "remove this flag" issue: we should keep this for 1 year.`